lib/streamlit/testing/v1/element_tree.py

```diff
@@ -998,14 +998,13 @@ def decrement(self) -> TimeInput:
 class Block:
     type: str
     children: dict[int, Node]
-    proto: BlockProto | None = field(repr=False)
+    proto: Any = field(repr=False)
```

Comment:
Because the other block classes store the message that lives within the BlockProto, so there is no more precise shared class. This mirrors how `Element` and its subclasses work. Making the type more precise with some refactoring would be possible but a pain